README.md

```diff
@@ -6,15 +6,46 @@ An experimental attempt at using [LDK](https://github.com/lightningdevkit/rust-l
 
 ## Setting up LNDK
 
+#### Compiling LND
+
 To run `LNDK`, `LND` is assumed to be running. For directions on how to do this, try [this guide](https://docs.lightning.engineering/lightning-network-tools/lnd/run-lnd).
 
-When compiling `LND`, make sure that the peersrpc and signerrpc services are enabled, like this:
+But note that when you get to the "install LND" section of the above guide, you'll need to make some adjustments to make `LND` compatible with `LNDK`.
+
+First, you'll need to run a particular [branch](https://github.com/lightningnetwork/lnd/tree/v0.16.2-patch-customfeatures) of `LND` to allow the advertising of onion_message feature bits. To do so, follow these steps:
```

Comment:
What do you think about switching up the order here a bit?
* LND needs to be run on branch 
* Compile with correct tags
* For full details see guide

Reason being that folks running LNDK will probably be more advanced, so we just need to point them to the correct branch/tags.